src/main/resources/assets/opencomputers/doc/de_DE/block/accessPoint.md

```diff
@@ -0,0 +1,11 @@
+# Access Point
+
+![AAA](oredict:oc:accessPoint)
+
+Der Access Point ist die drahtlose Variante des [Switches](switch.md). Dieser kann benutzt werden um Subnetzwerke zu separieren, so dass [Komponente](../general/computer.md) nicht in anderen Netzwerken erreichbar sind. Dennoch ist es möglich Nachrichten in andere Netzwerke zu schicken.
```

Comment:
`kann benutzt werden, um`
`zu separieren, sodass [Komponenten]`
`ist es möglich, Nachrichten in andere`
Do I really have to correct every single sentence?